src/Kernel-Tests-Extended/CompiledMethodTest.class.st

```diff
@@ -457,6 +457,24 @@ CompiledMethodTest >> testIsDeprecated [
 			ifFalse: [ self deny: (self class >> each) isDeprecated ] ]
 ]
 
+{ #category : #'tests - abstract' }
+CompiledMethodTest >> testIsExtension [
+
+	[
+	| method |
+	self class compile: 'isExtensionTestMethod ^ 1' classified: 'protocol'.
```

Comment:
Hi cyril 

do you know if the compile: adds to the epicea log?
Because I would like to avoid this. 

S.

Reply:
For now it does but I want to disable that globally for tests